src/go/k8s/tests/e2e/additional-configuration/verify-config-dev.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+echo "verify-config-v22.2.sh $*"
+retries=20
+until [ "$retries" -lt 0 ]; do
+  PANDAPROXY_RETRIES=$1
+  CRASH_LOOP_LIMIT=$2
+  if [ -z "$CRASH_LOOP_LIMIT" ] && [ -z "$PANDAPROXY_RETRIES" ]; then
+    echo "requires two argument, pandaproxy retries count and crash loop limit"
+    exit 1
+  fi
+  echo "Fetching configuration from $NAMESPACE/additional-configuration-0"
+  actual=$(kubectl -n "$NAMESPACE" exec additional-configuration-0 -- cat /etc/redpanda/redpanda.yaml)
+  expected=$(
+    cat <<EOF
+redpanda:
+    data_directory: /var/lib/redpanda/data
+    empty_seed_starts_cluster: false
+    seed_servers:
+        - host:
+            address: additional-configuration-0.additional-configuration.${NAMESPACE}.svc.cluster.local.
+            port: 33145
+    rpc_server:
+        address: 0.0.0.0
+        port: 33145
+    kafka_api:
+        - address: 0.0.0.0
+          port: 9092
+          name: kafka
+          authentication_method: none
+    admin:
+        - address: 0.0.0.0
+          port: 9644
+          name: admin
+    advertised_rpc_api:
+        address: additional-configuration-0.additional-configuration.${NAMESPACE}.svc.cluster.local.
+        port: 33145
+    advertised_kafka_api:
+        - address: additional-configuration-0.additional-configuration.${NAMESPACE}.svc.cluster.local.
+          port: 9092
+          name: kafka
+    developer_mode: true
+    auto_create_topics_enabled: true
+    cloud_storage_segment_max_upload_interval_sec: 1800
+    default_topic_partitions: 3
+    enable_idempotence: true
+    enable_rack_awareness: true
+    fetch_reads_debounce_timeout: 10
+    group_initial_rebalance_delay: 0
+    group_topic_partitions: 3
+    internal_topic_replication_factor: 3
+    log_segment_size: 536870912
+    log_segment_size_min: 1
```

Comment:
Please remove as v22.3.x doesn't have this property

Reply:
Good point.